src/main/webapp/app/shared/model/function.model.ts

```diff
@@ -0,0 +1,12 @@
+import { RealmDTO } from 'app/shared/model/realm.model';
+
+export interface IFunction {
```

Comment:
set the readonly default values

Reply:
is there a need for that? It's references as a read only array, so no modifications would be possible 
![image](https://user-images.githubusercontent.com/75810868/116690036-7f08d680-a9c1-11eb-9c09-5ce94e5d60b4.png)

Reply:
not sure you got this...here is link for the reference 
https://github.com/DX26-io/data-studio-react/blob/master/src/main/webapp/app/shared/model/user.model.ts
you shared the default value of array of function.i am referring to default values of followings
id: number;
  description: string;
  dimensionUse: boolean;
  measureUse: boolean;
  name: string;
  realm: RealmDTO;
  validation: string;
  value: string;